guidelines/v2.0/EXTENSIONS.md

```diff
@@ -1,21 +1,21 @@
-# Swagger Extensions
+# OpenAPI Extensions
 
-The Swagger 2.0 specification allows for custom properties to be added at several places within a Swagger definition, allowing
+The OpenAPI Specification version 2.0 allows for custom properties to be added at several places within an OpenAPI definition, allowing
 API providers to extend the meta-data provided for their REST APIs as needed. Extension properties are always 
 prefixed by "x-" and can have any valid JSON format value. 
 
 Currently extension properties are supported in the following definition objects:
   
-* within the [info object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#info-object)
-* within the [paths object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#paths-object)
-* within the [path-item object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#path-item-object)
-* within the [operation object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#operationObject) 
-* within the [parameter object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#parameterObject)
-* within the [responses object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#responses-object)
-* within the [tag object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#tag-object)
-* within the [security-scheme object](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#security-scheme-object)
+* within the [info object](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/2.0.md#info-object)
```

Comment:
should not need to include the branch - just use relative links instead of absolute links, such as :

`within the [info object](2.0.md#info-object)`